spec/providers/firestore.spec.ts

```diff
@@ -0,0 +1,22 @@
+import {expect} from 'chai';
+import fft = require('../../src/index');
+
+describe('providers/firestore', () => {
+  it('produces the right snapshot with makeDocumentSnapshot', async () => {
+    const test = fft();
```

Comment:
Nit - add a newline after this line and after you call makeDocumentSnapshot, to split up the arrange ,act, and assert steps of this test